sync/http/ready_test.go

```diff
@@ -0,0 +1,31 @@
+package http
+
+import (
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func Test_readyCheckRoute(t *testing.T) {
```

Comment:
could we rename it to `TestReadyCheckRoute` for consistency?

Reply:
Kept it that ways as there are already other tests following the same convention. You can have a look ok component_test, profiling_test, metrics_test and also the old health_test was following that same convention.